tests/suites/test_suite_ssl.function

```diff
@@ -3885,7 +3885,7 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS:MBEDTLS_SSL_PROTO_TLS1_3 */
+/* BEGIN_CASE depends_on:MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_TEST_HOOKS:MBEDTLS_SSL_PROTO_TLS1_3 */
```

Comment:
No need to depend on `USE_PSA_CRYPTO` here, we already depend on `MBEDTLS_SSL_PROTO_TLS1_3`.